index.js

```diff
@@ -12,11 +12,16 @@ module.exports = {
     }
   },
 
-  dynamicScript: function(request) {
-    var liveReloadPort = process.env.EMBER_CLI_INJECT_LIVE_RELOAD_PORT;
+  dynamicScript: function(options) {
+    var liveReloadOptions = options.liveReloadOptions;
+    if (liveReloadOptions.snipver === undefined) {
```

Comment:
it seems liveReloadOptions may be `undefined` at times: https://github.com/ember-cli/ember-cli-inject-live-reload/issues/35